Test/ServerTelemetryChannel.Test/Shared.Tests/SamplingTelemetryProcessorTest.cs

```diff
@@ -374,34 +385,73 @@ private static void TelemetryTypeDoesNotSupportSampling(Func<TelemetryProcessorC
             Assert.Equal(generatedCount, sentTelemetry.Count);
         }
 
-        private static void TelemetryTypeSupportsSampling(Action<TelemetryProcessorChain> sendAction, string excludedTypes = null, string includedTypes = null)
+        private static void TelemetryTypeSupportsSampling(Action<TelemetryProcessorChain> sendAction,
+                                                          string excludedTypes = null,
+                                                          string includedTypes = null)
+        {
+            TelemetryTypeSupportsSampling((chain) => { sendAction(chain); return 1; },
+                                          excludedTypes,
+                                          includedTypes);
+        }
+
+        private static void TelemetryTypeSupportsSampling(Func<TelemetryProcessorChain, int> sendAction,
+                                                          string excludedTypes = null,
+                                                          string includedTypes = null,
+                                                          int samplingPercentage = 10)
         {
             const int ItemsToGenerate = 100;
-            const int SamplingPercentage = 10;
             var sentTelemetry = new List<ITelemetry>();
-            var telemetryProcessorChainWithSampling = CreateTelemetryProcessorChainWithSampling(sentTelemetry, SamplingPercentage, excludedTypes, includedTypes);
+            TelemetryProcessorChain telemetryProcessorChainWithSampling = CreateTelemetryProcessorChainWithSampling(sentTelemetry,
+                                                                                                                    samplingPercentage,
+                                                                                                                    excludedTypes,
+                                                                                                                    includedTypes);
```

Comment:
It's just that now the parameters are on multiple lines and indented far to the right side so they make this diff look crazy.
The default StyleCop rules used to complain if the parameters were not either all on one line or all on their own lines which isn't the case here since one of them is still with the method call but personally I would just format it like before as all one line or put the parameters together on the second line with only a single indent, whatever you prefer.